src/Reestr/Rep/ResponseInterface.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Wearesho\Bobra\Ubki\Reestr\Rep;
+
+use Wearesho\Bobra\Ubki\Reestr;
+
+/**
+ * Interface ResponseInterface
+ *
+ * @package Wearesho\Bobra\Ubki\Reestr\Rep
+ */
+interface ResponseInterface extends Reestr\ResponseInterface
+{
+    /**
+     * Reestr type
+     *
+     * @return string
+     */
+    public function getErtype(): string;
```

Comment:
Is error always exists?

Reply:
This method return type of reestr. For error type have method getCrytical(). And this attr always exist